sqlglot/dialects/materialize.py

```diff
@@ -0,0 +1,58 @@
+from sqlglot import exp
+from sqlglot.dialects.dialect import NormalizationStrategy
+from sqlglot.dialects.postgres import Postgres
+from sqlglot.tokens import TokenType
+
+from sqlglot.transforms import (
+    remove_unique_constraints,
+    ctas_with_tmp_tables_to_create_tmp_view,
+    preprocess,
+)
+
+
+class Materialize(Postgres):
```

Comment:
Why does this inherit from Postgres?

Reply:
I guess the dialect is largely based on Postgres? Just saw https://github.com/MaterializeInc/materialize?tab=readme-ov-file#transform-manipulate-and-read-your-data

Reply:
Yep indeed, Materialize is [Postgres wire-compatible](https://materialize.com/blog/postgres-compatibility/).